packages/rs-dapi-client/src/dapi_client.rs

```diff
@@ -114,12 +132,17 @@ impl DapiRequestExecutor for DapiClient {
         TransportError: Mockable,
     {
         // Join settings of different sources to get final version of the settings for this execution:
-        let applied_settings = self
+        let mut applied_settings = self
             .settings
             .override_by(R::SETTINGS_OVERRIDES)
             .override_by(settings)
```

Comment:
Shall we just call it here and if it's None it won't override anything?

Reply:
refactored